### ISSUE TITLE
How to correctly use shtab

### ISSUE THREAD

**Issue by JakobDev** [iterative/shtab#132]
shtab looks like a nice thing that I want to add to my programs. But there's one big question: How can I use shtab, so It's useable for end users? The [documentation](https://docs.iterative.ai/shtab/use/) says, that end users can actually using the shtab cli or by adding a new cli option provided by shtab to the Program. But that is not very useful at all. The User first needs to know that this can be done. Putting it in some documentation where almost nobody reads it. And the User also needs to add it to the` .bashrc` or `.zshrc` to make it work on startup.

The completion should work out of the Box after installing. A solution would be to put the files in the specific directories (e.g. usr/share/bash-completion/completions), but this is not supported by pip.

**Comment by casperdcl** [comment]
Duplicate of #122?
In particular, the end-user docs link at the bottom of https://github.com/iterative/shtab/issues/122#issuecomment-1333968449 might be helpful (related to #199).

**Comment by LimbersMay** [comment]
That's true, this is one of the worst documentations I've read, it doesn't even specify how to make it all work
